src/components/WebCLI/WebCLI.tsx

```diff
@@ -127,22 +127,53 @@ const WebCLI = ({
     const conn = new Connection({
       address: wsAddress,
       onopen: () => {
-        // Unused handler.
+        // Close this websocket connection if it was in CONNECTING state
+        // while a new websocket connection was established.
+        if (conn.websocket !== connection.current?.websocket) {
+          conn.disconnect();
+        }
```

Comment:
This can be removed as we're not creating a new connection if there is an active one, and in strict dev mode the socket will get closed by the cleanup useEffect.

```suggestion
        // Unused handler.
```